build.gradle.kts

```diff
@@ -40,7 +40,7 @@ dependencies {
     api("io.netty:netty-all:4.1.87.Final")
     api("com.google.protobuf:protobuf-java:3.21.12")
 
-    implementation(kotlin("stdlib"))
+    implementation(kotlin("stdlib-jdk8"))
```

Comment:
I believe no need to revert this lib back to `stdlib-jdk8` as we switched to Java-11 target

Reply:
From what I read from Kotlin 1.8 [stdlib-jdk8 is merged into kotlin-stdlib](https://kotlinlang.org/docs/whatsnew18.html#updated-jvm-compilation-target) . Otherwise, it is preferred to use `stdlib-jdk8` to make use of the jdk8 extensions.